### PR TITLE
renamed variable ZZ -> zz in the tested documentation examples

### PR DESCRIPTION
**Pull request by mohamed-barakat**
see https://github.com/gap-system/PackageDistro/pull/821#issuecomment-1702262330

### CODE REVIEW

File: MatricesForHomalg/gap/Integers.gi [hunk 1]
```diff
@@ -216,4 +216,4 @@ InstallMethod( CreateHomalgTable,
 end );
 
 ## create a globally defined ring of integers
-HOMALG_MATRICES.ZZ := HomalgRingOfIntegers( );
+HOMALG_MATRICES.zz := HomalgRingOfIntegers( );
```
Comment (zickgraf) [inline MatricesForHomalg/gap/Integers.gi:219]: Is this change deliberate? If yes, two things:
1. This will break ToricVarieties (just grep for `HOMALG_MATRICES.ZZ` in ToricVarieties_project)
2. For consistency, `HOMALG_MATRICES.QQ` should be renamed, too.

Comment (mohamed-barakat) [reply MatricesForHomalg/gap/Integers.gi]: > 1. This will break ToricVarieties (just grep for `HOMALG_MATRICES.ZZ` in ToricVarieties_project)

Was not intended. I was in a hurry in the morning and probably made a mistake in the regular expression.

> 2. For consistency, `HOMALG_MATRICES.QQ` should be renamed, too.

See above.

Comment (zickgraf) [reply MatricesForHomalg/gap/Integers.gi]: Ah, I see, then everything's fine :-) I was a bit worried that you were planning larger changes :D

Comment (mohamed-barakat) [reply MatricesForHomalg/gap/Integers.gi]: I wanted a minimal change. I am unhappy with these changes.

Comment (zickgraf) [reply MatricesForHomalg/gap/Integers.gi]: > I am unhappy with these changes.

Me too. The suggestion that we should only use names starting with lowercase letters would make everything much worse to read.... :/
